dev-tools/Dockerfile

```diff
@@ -0,0 +1,28 @@
+
+FROM ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+
+RUN apk add python3 \
+            py3-qt5 \
+            tesseract-ocr \
+            qemu-arm \
+            libusb
+
+ARG QEMU_BIN=qemu-arm
+
+# -static version is not available in Alpine repositories
+RUN ln -s "$QEMU_BIN" $(dirname $(which "$QEMU_BIN"))/$QEMU_BIN-static
+
+# Python packages building dependencies, can be removed afterwards
+ARG PYTHON_BUILD_DEPS=python3-dev,musl-dev
+
+# Install the building dependencies.
+RUN apk add $(echo -n "$PYTHON_BUILD_DEPS" | tr , ' ')
+
+# Install test tools (Ragger framework, Speculos emulator, Ledgerblue...)
+RUN pip3 install --extra-index-url https://test.pypi.org/simple/ ragger[tests,all_backends]
+
+# Cleanup, remove packages that aren't needed anymore
+RUN apk del $(echo -n "$PYTHON_BUILD_DEPS" | tr , ' ')
+
+# Dependency for app-boilerplate python functional tests
+RUN pip3 install pysha3
```

Comment:
shall it be in the docker on installed at runtime as will other lib be?

Reply:
@xchapron-ledger my opinion is that it should be installed with `pip3 install -r tests/requirements.txt` of each app (even app-boilerplate) we even have a vscode action for that in the new app-boilerplate PR : 

```
            "label": "Install tests requirements",
            "type": "shell",
            // Installs functional tests python requirements in the docker container. 
            "command":"docker exec -it ${config:container_name} bash -c 'pip3 install -r tests/requirements.txt'",
```

@tdejoigny-ledger insists it should be installed via the dockerfile.

Reply:
@agrojean-ledger and @xchapron-ledger yes it is true, I asked to add this dependency 😄 The intent here was to have a docker usable as it is to build & run the tests for boilerplate without any extra "pip install". This exception is only for Boiler plate. 

I can change my mind if anybody think that it is a bad idea 😜

Reply:
We will have to maintain these dependencies if there are any such changes in boilerplate app. This is one reason not to do that. In general I think it would be a bad idea to tailor this container for boilerplate as it is intended to be a general purpose image for development.

What's more it costs almost nothing to the user to follow one more step in the quick start guide to install the requirements with the vscode task we provide.

Reply:
Ok you can remove the dependency if this extra step is mentioned in quick start guide.